### DISCUSSION TITLE
Sign Text

### DISCUSSION THREAD

**Discussion by Lardbutt200** [maruohon/litematica#750] [Q&A]
Every Schematic i download that i know should have text on the signs renders, as well as pastes blank. I dont think its just an NBT pasting issue seeing as the render is also blank which didnt use to be the case. Currently running fabric for 1.20.1 as well as litematica for 1.20.1-0.15.0

**Comment by maruohon** [comment]
Update to 0.15.3. It fixes this at least partially, by adding a converter from the old sign block entity NBT format to the 1.20 format. I've still seen reports of it not working for some people, so I must have missed something or gotten something wrong with that converter, I think...

**Reply by Lardbutt200** [reply]
Thank you, will try :)
